app/controllers/weets.js

```diff
@@ -0,0 +1,6 @@
+const weetService = require('../services/weets');
+
+exports.getQuote = async (_, res) => {
+    const data = await weetService.getRandomQuote();
+    res.status(200).send(data)
+};
```

Comment:
We don't actually need a REST endpoint for this feature, so this controller is not needed

Reply:
For future endpoints remember to use the informational logs, this makes API debugging much easier.